opentelemetry-user-events-logs/examples/README.md

```diff
@@ -0,0 +1,100 @@
+# Getting Started with OpenTelemetry Rust Logs in 5 Minutes - Exporting logs to user_events
+
+## Prerequisites
+
+* Rust and Cargo: Install them if you haven't.
+* A Linux system: Make sure it supports user_events.
+* Correct Permissions: Linux user(preferable non-root) with write permission to the tracefs.
+* A listener/agent to listen for new events. Or else `perf` and `decode-perf` tools installed for validation.
+
+## Steps
+1. Create a new Rust application and run it.
+
+```sh
+$ cargo new otel-userevents-getting-started
+$ cd otel-userevents-getting-started
+$ cargo run
+```
+
+You'll see "Hello, world!".
+
+2. Open the `Cargo.toml` file and paste these lines
+```toml
+[dependencies]
+opentelemetry_sdk = "0.20.0"
+opentelemetry-appender-tracing = "0.1.0"
+opentelemetry-user-events-logs = "0.1.0"
+tracing = { version = "0.1.37", default-features = false, features = ["std"] }
+tracing-core = "0.1.31"
+tracing-subscriber = { version = "0.3.0", default-features = false, features = ["registry", "std"] }
+
+```
+
+3. Replace what's inside `src/main.rs` with the example file [basic.rs](./basic.rs). This new code sets up logging, make new tracepoints, and send an error message where it needs to go.
+4. Run and Check: Go back to the terminal and type `cargo run`. If everything is okay, your logs will be sent to the proper place. To test this, you can use the `perf` and `decode-perf` tool like this:
+
+```sh
+$ perf record -e user_events:test_L2K1Gtest
+<ctrl-c after running sample code>
+$ decode-perf ./perf.data
+<log data is output here>
+```
+
+Note: The sample code sets up a tracepoint called `test_L2K1Gtest`, and error message are relayed to this tracepoint. Refer to next section for more details.
```

Comment:
1. Change "sample" to "example" ("sample" has many different meanings in the context of OpenTelemetry).
2. What does "error message" and "relay" mean?

Reply:
Changed to "error log is exported to this tracepoint", along with other comments.